tests/test_data.py

```diff
@@ -403,7 +403,7 @@ async def test_agent_process(init_dataclasses_and_load_schema):
     agi2 = data.AgentInstance(process=agent_proc.sid, name="agi2", tid=env.id)
     await agi2.insert()
 
-    agent_procs = await data.AgentProcess.get_by_env(env=env.id)
+    agent_procs = await data.AgentProcess.get_list(environment=env.id, order_by_column="last_seen", order="ASC NULLS LAST")
```

Comment:
This means we are breaking compatibility with with respect to sorting?

Reply:
No, why? This change comes only from the fact that `get_by_env` was only used here, so to remove `get_by_env` I had to replace its call here by its implementation in this case.

Reply:
ok